src/templates/Profile.tsx

```diff
@@ -771,220 +960,180 @@ const Profile = () => {
+                        //       );
+                        //     }}
+                        //   >
+                        //     <OutlinedButton>
+                        //       Copy Verification Text
+                        //     </OutlinedButton>
+                        //   </div>
+                        // ),
+                        button2: 'ok',
+                      })
+                    }
+                  />
+                </BadgeCard>
               </div>
-              <div className="my-2 flex items-center">
-                {nip05 ? (
-                  <>
-                    <a
-                      href={`https://${
-                        nip05.split('@')[1]
-                      }/.well-known/nostr.json?name=${nip05.split('@')[0]}`}
-                      target="_blank"
-                      rel="noreferrer"
-                    >
-                      <AlternateEmail
-                        color="success"
-                        className="mr-2"
-                        fontSize="large"
-                      />
-                    </a>
-                    <span>
-                      User has a valid NIP-05 identifier: <b>{nip05}</b>
-                    </span>
-                  </>
-                ) : (
-                  <>
-                    <AlternateEmail
-                      color="error"
-                      className="mr-2"
-                      fontSize="large"
-                    />
-                    <span>
-                      User has <b>NOT</b> set a NIP-05 identifier for
-                      themselves.
-                    </span>
-                  </>
-                )}
-                <HelpOutline
-                  className="cursor-pointer !ml-1 align-middle"
-                  onClick={() =>
-                    setDialog({
-                      open: true,
-                      title: 'NIP-05 Verification',
-                      text: (
-                        <>
-                          User is expected to; <br />
-                          Have a nip05 tag on their nostr profile that resolves
-                          to their Hex Public Key as per{' '}
-                          <a
-                            href="https://github.com/nostr-protocol/nips/blob/master/05.md"
-                            target={'_blank'}
-                            rel="noreferrer"
-                            className="!underline"
-                          >
-                            NIP-05 nostr specification.
-                          </a>
-                        </>
-                      ),
-                      button1: <></>,
-                      button2: 'ok',
-                    })
+
+              {/* Add information for github */}
+              <div className="my-2 flex items-center w-full md:w-max">
+                <BadgeCard
+                  variant={'github'}
+                  verified={!!tweet.telegram}
```

Comment:
@ng5jr  github badge is checking telegram. this should check for githubIdentity which is a local check and then popup shows telegram but it should show github verification instructions.